src/test/java/reposense/parser/AuthorConfigParserTest.java

```diff
@@ -0,0 +1,178 @@
+package reposense.parser;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import reposense.model.Author;
+import reposense.model.AuthorConfiguration;
+import reposense.model.RepoLocation;
+
+public class AuthorConfigParserTest {
+    private static final String TEST_REPO_BETA_LOCATION = "https://github.com/reposense/testrepo-Beta.git";
+    private static final String TEST_REPO_BETA_BRANCH = "master";
+
+    private static final Path AUTHOR_CONFIG_EMPTY_LOCATION_FILE = new File(AuthorConfigParserTest.class.getClassLoader()
```

Comment:
Since we are already splitting `CsvParserTest`, there is no reason not to split its corresponding resource folder.

Reply:
Sorry I am not sure if I understand you correctly. The above were used in tests related to the author configs, same for group. So I was hesitant to remove them.

Reply:
Hi, what he meant was to separate the test resources into separate folders for author & group etc, instead of storing it under `CsvParserTest`.

https://github.com/reposense/RepoSense/blob/d865587704b0e0b7e3d45849ef3bdfd4bf40eb71/src/test/java/reposense/parser/AuthorConfigParserTest.java#L25

Reply:
@jamessspanggg Noted.

Reply:
Could you follow the OSS format of the commit message body as well? This is to standardise and ease current/future developers in understanding the work done in PRs. 

<img width="985" alt="Screen Shot 2019-12-14 at 12 06 21 PM" src="https://user-images.githubusercontent.com/32864116/70843248-27200d00-1e6a-11ea-9aff-1c98b9c964db.png">